src/attributes.jl

```diff
@@ -488,6 +488,12 @@ struct ConstraintDualStart <: AbstractConstraintAttribute end
 
 The assignment to the constraint primal values in result `N`.
 If `N` is omitted, it is 1 by default.
+
+Given a constraint `function-in-set`, the `ConstraintPrimal` is the value of the
+function evaluated at the primal solution of the variables. For example, given
+the constraint `ScalarAffineFunction([x,y], [1, 2], 3)`-in-`LessThan(0)` and 
```

Comment:
Maybe an example with `LessThan(4)` would be better to show that this value is not used in the constraint primal.